tests/integration/lsp_tests.rs

```diff
@@ -11786,6 +11786,45 @@ fn lsp_jupyter_byonm_diagnostics() {
   client.shutdown();
 }
 
+#[test]
+fn lsp_deno_future_env_byonm() {
+  let context = TestContextBuilder::for_npm().use_temp_cwd().build();
+  let temp_dir = context.temp_dir();
+  temp_dir.write("package.json", json!({}).to_string());
+  let mut client = context.new_lsp_command().env("DENO_FUTURE", "1").build();
+  client.initialize_default();
+  let diagnostics = client.did_open(json!({
+    "textDocument": {
+      "uri": temp_dir.uri().join("file.ts").unwrap(),
+      "languageId": "typescript",
+      "version": 1,
+      "text": "import \"npm:chalk\";\n",
+    },
+  }));
+  assert_eq!(
```

Comment:
Can you update the test for check for a package that is actually listed in `package.json`?